cmd/juno/juno.go

```diff
@@ -159,6 +162,7 @@ func NewCmd(config *node.Config, run func(*cobra.Command, []string) error) *cobr
 	junoCmd.Flags().Bool(p2pF, defaultP2p, p2pUsage)
 	junoCmd.Flags().String(p2pAddrF, defaultP2pAddr, p2PAddrUsage)
 	junoCmd.Flags().String(p2pBootPeersF, defaultP2pBootPeers, p2pBootPeersUsage)
+	junoCmd.Flags().Uint16(metricsPortF, defaultMetricsPort, metricsPortUsage)
```

Comment:
I think it would be best to have a `bool` flag where by default we don't have metrics enabled and if the user wants to enable it we use port `9090` as that is the expected Prometheus metrics port. I highly doubt someone would want to run it on something other than 9090, if someone does request it then we can make it configurable.

Reply:
done